web/satellite/src/components/account/billing/paymentMethods/StripeInput.vue

```diff
@@ -34,7 +34,7 @@ export default class StripeInput extends Vue {
             return;
         }
 
-        this.stripe = window['Stripe'](process.env.VUE_APP_STRIPE_PUBLIC_KEY);
+        this.stripe = window['Stripe'](this.$store.state.appStateModule.stripePublicKey);
```

Comment:
lets wrap into a class that works with stripe library

Reply:
Agree with @rikysya